### PR TITLE
Add tests for all remaining treeview examples

### PR DESCRIPTION
**Pull request by spectranaut**
- [treeview/treeview-1/treeview-1b.html](https://w3c.github.io/aria-practices/examples/treeview/treeview-1/treeview-1b.html)
- [treeview/treeview-2/treeview-2a.html](https://w3c.github.io/aria-practices/examples/treeview/treeview-2/treeview-2a.html)
- [treeview/treeview-2/treeview-2b.html](https://w3c.github.io/aria-practices/examples/treeview/treeview-2/treeview-2b.html)

### CODE REVIEW

**Comment by mcking65** [comment]
@jessebeach, @sh0ji, @tatermelon, this is ready for review.

File: test/tests/treeview-2a.js [hunk 1]
```diff
@@ -0,0 +1,723 @@
...
+  let closedFolders = await t.context.session.findElements(By.css(closedFoldersSelector));
+
+  // Going through all closed folder elements in dom order will open parent
+  // folders first, therefore all child folders will be visible before clicked
+  for (let folder of closedFolders) {
+    await folder.click();
+  }
+};
+
+const checkFocus = async function (t, selector, index) {
```
Comment (jessebeach) [inline test/tests/treeview-2a.js]: This is defined in two files (or more, I noticed 2). Does it make sense to consolidate these implementations in a followup?

Comment (spectranaut) [reply test/tests/treeview-2a.js]: I think you are right, this should be broken out.

**Comment by spectranaut** [comment]
I'm going to rebase these commits on top of the the other changes I've submitted to bocoup in order for us to see passing checks :)

**Pull request merged**